app/authentication/map_role_to_permissions.py

```diff
@@ -0,0 +1,12 @@
+from app import settings
+
+
+def map_role_to_permissions(roles):
```

Comment:
Not sure if we need roles and permissions at this moment in time since we only have one role with one permission. If we simplify that this can be removed.